fava/api/__init__.py

```diff
@@ -90,6 +90,19 @@ def _sidebar_links(entries):
             for entry in sidebar_link_entries]
 
 
+def _settings(entries):
+    """ Parse custom entries for fava settings. They have the following format:
+
+    2016-04-01 custom "fava-option" "[setting_name]" [setting_value]
+
+    """
+    settings_entries = [
+        entry for entry in _filter_entries_by_type(entries, Custom)
```

Comment:
Maybe filter out the custom entries in `BeancountReportAPI` and store them in `self.custom_entries` there. We now have three functions that only need custom entries, but run over all entries to filter them out.